internal/runners/export/log.go

```diff
@@ -57,12 +60,24 @@ func (l *Log) Run(params *LogParams) (rerr error) {
 	if err != nil {
 		return ErrInvalidLogPrefix
 	}
+
 	for _, file := range logFiles {
 		if regex.MatchString(file) {
 			timestamp, err := strconv.Atoi(regex.FindStringSubmatch(file)[1])
 			if err != nil {
 				continue
 			}
+
+			ignore, err := ignoreLogFile(file)
+			if err != nil {
+				logging.Error("failed to validate log file: %s", err.Error())
+				continue
```

Comment:
Bubble up errors please. This shouldn't ever cause an error, so if it does something is broken and we need to fix it. Logging just obfuscates the breakage.